src/shaders/doubleSphereCamera.frag

```diff
@@ -35,8 +35,6 @@ out highp uint fragmentObjectId;
 void main(void) {
   vec3 m;
   m.xy = (gl_FragCoord.xy - PrincipalPointOffset) / FocalLength;
-  // MUST flip the x axis
-  m.x = -m.x;
```

Comment:
Like I said, the difference between the "old and new" cubemaps is 180 degrees around Y-axis.
The orientation of the "new" cubemap is actually the "correct" one we should adopt. (+Z points inwards)

Before I directly flipped the x which does 2 things:
- convert the coordinates from right-handed to left handed.
- rotate the frame round Y-axis by 180 degrees (making the +Z from outwards to inwards direction)
That is why the results are consistent and correct "before and after" this change.